tests/core/eth-module/test_eth_protocolVersion.py

```diff
@@ -0,0 +1,2 @@
+def test_eth_protocolVersion(web3):
+    assert web3.eth.protocolVersion == '99'
```

Comment:
Why is this 99?

Reply:
I believe it's talking to `eth-tester` and should probably be updated internally to be `63`

Reply:
Hmmm, strange - according to [this line](https://github.com/ethereum/web3.py/blob/master/web3/providers/eth_tester/defaults.py#L165) it should be returning `63`. Will dive deeper into this tomorrow